plugins/workspace-minimap/src/positioned_minimap.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A positionable version of the minimap.
+ * @author cesarades@google.com (Cesar Ades)
+ */
+
+import * as Blockly from 'blockly/core';
+import {Minimap} from './minimap';
+
+/**
+ * A positionable version of minimap that implements IPositionable.
+ * @param {Minimap} minimap The minimap to position.
+ * @implements {Blockly.IPositionable}
```

Comment:
I would remove this, and then use `export class PositionedMinimap implements Blockly.IPositionable` when you get to that point.

Reply:
Yea will do when I get to work on that. If I add it now it gives me errors since I dont have the necessary methods.